File: 1823-determine-if-string-halves-are-alike/determine-if-string-halves-are-alike.py
class Solution(object):
    def halvesAreAlike(self, s):
        """
        :type s: str
        :rtype: bool
        """
        a=""
        b=""
        c=0
        d=0
        for i in range(len(s)/2):
            a+=s[i]
        for i in range(len(s)/2,len(s)):
            b+=s[i]
        for i in a:
            if i in "aeiouAEIOU":
                c+=1
        for i in b:
            if i in "aeiouAEIOU":
                d+=1
        if(c==d):
            return True
        else:
            return False

File: 1823-determine-if-string-halves-are-alike/determine-if-string-halves-are-alike.cpp
class Solution {
public:
    bool halvesAreAlike(string s) {
        int n=s.size(),a=0,b=0;
        for(int i=0;i<(n/2);i++)
        {
            if(s[i]=='a' or s[i]=='e' or s[i]=='i' or s[i]=='o' or s[i]=='u' or s[i]=='A' or s[i]=='E' or s[i]=='I' or s[i]=='O' or s[i]=='U')
            {
                a++;
            }
        }
        for(int i=n/2;i<n;i++)
        {
            if(s[i]=='a' or s[i]=='e' or s[i]=='i' or s[i]=='o' or s[i]=='u' or s[i]=='A' or s[i]=='E' or s[i]=='I' or s[i]=='O' or s[i]=='U')
            {
                b++;
            }
        }
        if(a==b) return true;
        return false;    
    }
};